app/src/main/java/com/github/leuludyha/ibdb/presentation/screen/profile/UserProfile.kt

```diff
@@ -62,17 +61,20 @@ fun UserProfile(
         Column(
             modifier = Modifier.fillMaxWidth()
         ) {
-            Button("My Friends")
-            Button("Something")
-            Button("Settings")
+            Button(Constant.USER_PROFILE_MY_FRIENDS)
+            Button(Constant.USER_PROFILE_RECEIVE_WORK) {
+                // Start listening for people sharing other works
+                navController.navigate(Screen.ReceiveNearbyWork.route)
+            }
+            Button(Constant.USER_PROFILE_SETTINGS)
         }
     }
 }
 
 @Composable
-private fun Button(text: String){
+private fun Button(text: String, onClick: () -> Unit = {}) {
```

Comment:
It's maybe a bit confusing to have it named `Button`, giving a more insightful name to it could be useful

Reply:
I actually don't fully understand what the purpose of that function was, but I hesitated to remove it. At the end of the day, it would be almost identical to just use the jetpack compose button.